packages/synapse-react-client/src/components/SetBasicAccessRequirementFields/AccessRequirementTextInstructions.tsx

```diff
@@ -28,37 +36,48 @@ export const AccessRequirementTextInstructions: React.FunctionComponent<
           <Typography variant="body1" fontWeight={700} mb={1}>
             Legacy text-only instructions
           </Typography>
-          <Typography variant="body1" mb={1}>
-            {oldInstructions}
-          </Typography>
-          <Button
-            variant="contained"
-            color="error"
-            onClick={() => setOpen(true)}
-          >
-            Delete old instructions
-          </Button>
-          <ConfirmationDialog
-            open={open}
-            title="Are you sure?"
-            content="Deleting the old instructions cannot be undone. Continue?"
-            onCancel={() => setOpen(false)}
-            onConfirm={() => {
-              const clearedInstructionsAr = { ...accessRequirement }
-              if (
-                clearedInstructionsAr.concreteType ===
-                TERMS_OF_USE_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE
-              ) {
-                clearedInstructionsAr.termsOfUse = undefined
-              } else if (
-                clearedInstructionsAr.concreteType ===
-                ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE
-              ) {
-                clearedInstructionsAr.actContactInfo = undefined
-              }
-              onConfirmDelete(clearedInstructionsAr)
-            }}
-          />
+          {isDeleting ? (
+            <Box mb={1}>
+              <SynapseSpinner />
+            </Box>
+          ) : (
+            <Typography variant="body1" mb={1}>
+              {oldInstructions}
+            </Typography>
+          )}
```

Comment:
Show spinner when deleting in this component, rather than managing the spinner in the parent